### DISCUSSION TITLE
rAudio 1 • 20210827: settings almost unreachable on my smartphone

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#491] [Fixed Issues]
On my smartphone (HTC U11) clicking the cogwheel in the playback window results almost always in the "Library Home Display" settings and not in the settings menu. "Almost always" means: If I click slightly north of the cogwheel I get the power-off/reboot menu. 
That's the behaviour using Chrome. If I switch to Firefox then the settings menu can be seen for a short split second, than presenting the "Library Home Display".
Workaround: Using the phone in landscape mode.

**Comment by jay77777777** [comment]
Noticed this right away after the 8-27 update.

**Reply by AndyFx12** [reply]
Same here.

**Comment by klassikmann** [comment]
**rAudio 1 • 20210903**
Problems still present. Some menu entries require landscape mode, some others portrait mode. 
Examples on request.

**Reply by rern** [reply]
Please try update again.

**Comment by klassikmann** [comment]
Fixes in rAudio 1 • 202109032
Thank you
